spec/unit/my_instance_methods_spec.rb

```diff
@@ -96,6 +96,129 @@
           expect(subject['emb_threes']).to be_nil
         end
       end
+
+      context 'when embeds_one object blank' do
+        let(:emb_one) { nil }
+
+        it 'should not include embeds_one model key' do
+          expect(subject.keys).to_not include 'emb_one'
+        end
+      end
+
+      describe 'embeds_one' do
+        before(:all) do
```

Comment:
I used `let` for latest commits wherever there was no need of class name constant